diagnostics-app/package.json

```diff
@@ -66,6 +70,7 @@
     "@angular/compiler-cli": "~12.0.2",
     "@types/jasmine": "~3.6.0",
     "@types/node": "^12.11.1",
+    "husky": "^6.0.0",
```

Comment:
How does it affect development workflow, do i need to do some additional steps for setup? I think I should run some commands to setup git hooks locally, can you please document it?

Reply:
We don't need to anything extra to set this up. I have added a `postinstall` script that initialises husky automatically!

Reply:
Not needed any more. I've documented the instructions to set up git hooks in README.md